From e89d67596e202119ea846cc997f4cf75cb284490 Mon Sep 17 00:00:00 2001
From: Yijing Wang <wangyijing@huawei.com>
Date: Fri, 13 Oct 2017 16:35:32 -0700
Subject: [PATCH] bcache: Remove redundant set_capacity
Git-commit: e89d67596e202119ea846cc997f4cf75cb284490
Patch-mainline: v4.15-rc1
References: bsc#1076110

set_capacity() has been called in bcache_device_init(),
remove the redundant one.

(Coly Li: rebased for SLE12-SP2)

Signed-off-by: Yijing Wang <wangyijing@huawei.com>
Reviewed-by: Eric Wheeler <bcache@linux.ewheeler.net>
Acked-by: Coly Li <colyli@suse.de>
Signed-off-by: Jens Axboe <axboe@kernel.dk>
Signed-off-by: Coly Li <colyli@suse.de>

---
 drivers/md/bcache/super.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/md/bcache/super.c
+++ b/drivers/md/bcache/super.c
@@ -1142,9 +1142,6 @@ static int cached_dev_init(struct cached
 	if (ret)
 		return ret;
 
-	set_capacity(dc->disk.disk,
-		     dc->bdev->bd_part->nr_sects - dc->sb.data_offset);
-
 	dc->disk.disk->queue->backing_dev_info.ra_pages =
 		max(dc->disk.disk->queue->backing_dev_info.ra_pages,
 		    q->backing_dev_info.ra_pages);
